<?xml version="1.0"?>
<launch>
    <param name="mesh_topic" value="map_mesh" />
    <param name="raytrace_frame" value="world" />
    <node name="raytracer_service" pkg="cgal_raytracer" 
          type="raytracer_service" output="screen" />
</launch>
